src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html

```diff
@@ -0,0 +1,101 @@
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Parking brake information</dt>
+    <dd id="psv-brakes-dataTrBrakeThree" class="govuk-summary-list__value">
+      {{ brakes?.dataTrBrakeThree| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Retarder brake one</dt>
+    <dd id="psv-brakes-retarderBrakeOne" class="govuk-summary-list__value">
+      {{ brakes?.retarderBrakeOne| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Retarder brake two</dt>
+    <dd id="psv-brakes-retarderBrakeTwo" class="govuk-summary-list__value">
+      {{ brakes?.retarderBrakeTwo| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <br>
+  <h2 class="govuk-heading-m">Brake force wheels not locked</h2>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Service</dt>
+    <dd id="psv-brakes-serviceBrakeForceA" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsNotLocked?.serviceBrakeForceA| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Secondary</dt>
+    <dd id="psv-brakes-secondaryBrakeForceA" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsNotLocked?.secondaryBrakeForceA| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Parking</dt>
+    <dd id="psv-brakes-parkingBrakeForceA" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsNotLocked?.parkingBrakeForceA| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <br>
+  <h2 class="govuk-heading-m">Brake force wheels up to half locked</h2>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Service</dt>
+    <dd id="psv-brakes-serviceBrakeForceB" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsUpToHalfLocked?.serviceBrakeForceB| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Secondary</dt>
+    <dd id="psv-brakes-secondaryBrakeForceB" class="govuk-summary-list__value">
+      {{ brakes?.brakeForceWheelsUpToHalfLocked?.secondaryBrakeForceB| DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Parking</dt>
+    <dd id="psv-brakes-parkingBrakeForceB" class="govuk-summary-list__value">
```

Comment:
needs to be prepended by test-* if this is required by automation testers, please check with Dragos